src/platforms/esp32/components/avm_builtins/i2c_driver.c

```diff
@@ -82,10 +82,16 @@ static const AtomStringIntPair cmd_table[] = {
     SELECT_INT_DEFAULT(I2CInvalidCmd)
 };
 
+static const char *const scl_io_num_atom = ATOM_STR("\xA", "scl_io_num");
```

Comment:
I think now there is little value in defining them here, just leave them in place, such as:
```c
term scl_io_num_term = interop_kv_get_value(opts, ATOM_STR("\xA", "scl_io_num"), global);
```

Reply:
A bit hard to do that, if it's being used in a macro, as ☝🏽

Reply:
Doing this doesn't really add any value and in general makes the code harder to read.